docs/source/contributing.rst

```diff
@@ -37,12 +37,12 @@ latest local version.
 
 Notes:
 
-1. Any changes to the template will need to be committed (not pushed) to be
+#. Any changes to the template will need to be committed (not pushed) to be
 picked up by Copier.
-2. If there's an opportunity for introducing an automated test, please take it.
-3. This project has some automated testing, to ensure that the template can
+#. If there's an opportunity for introducing an automated test, please take it.
+#. This project has some automated testing, to ensure that the template can
 hydrate a reasonable output project - feel free to extend that if it's reasonable.
-4. See the `Copier documentation <https://copier.readthedocs.io/en/latest/generating/#regenerating-a-project>`_
+#. See the `Copier documentation <https://copier.readthedocs.io/en/latest/generating/#regenerating-a-project>`_
```

Comment:
Probably a silly question, but did you build the docs locally to check that this renders correctly? I ask only because when I "View file" in the github UI, the `#` are just rendered as `#`, instead of a numbered list.